colin/core/ruleset/loader.py

```diff
@@ -42,7 +42,7 @@ def get_ruleset_struct_from_file(file_path):
     except ColinRulesetException as ex:
         raise ex
     except Exception as ex:
-        msg = "Ruleset '{}' cannot be loaded: {}".format(file_path, ex)
```

Comment:
Use % formatting in logging functions and pass the % parameters as arguments instead of f-string

Reply:
nvm; msg is used also in te exception